RDKShell/RDKShell.cpp

```diff
@@ -1509,6 +1509,13 @@ namespace WPEFramework {
             return make_shared<WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>>("Packager.1", "", false, query);
         }
 
+        std::shared_ptr<WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>> RDKShell::getLISAPlugin()
```

Comment:
this violates `7. Inter-plugin communication` of the [readme](https://github.com/rdkcentral/rdkservices#readme)

Reply:
@npoltorapavlo , updated the call to LISA to use JSONRPCDirectLink and thus QueryInterfaceByCallsign()